### PR TITLE
Masks are now logged.

### LINKED ISSUE #18271: Masks of transmogrification aren't logged
#### Description of issue

There seems to be no logs for masks of transmogrification being equipped in the code. Which is an issue, as wearing those masks can transform you into what looks like an antagonist and can mean a lot of trouble, chaos, and ahelps.
It should have a log/message for admins.
Reported in the thread.

### PR DESCRIPTION
**Pull request by ShiftyRail**
Resolves #18271 
proof

### CODE REVIEW

File: code/modules/clothing/masks/transmog.dm [hunk 1]
```diff
@@ -43,6 +43,10 @@
 					C.transmogrify(target_type)
 				else
 					C.transmogrify(target_type, revert_spell_type || TRUE)
+				var/turf/T = get_turf(C)
```
Comment (MadmanMartian) [inline code/modules/clothing/masks/transmog.dm]: Put it above the transmogrifies so it gives the key_name properly, maybe?

Comment (ShiftyRail) [reply code/modules/clothing/masks/transmog.dm]: Yes this is a good idea

**Comment by jknpj** [comment]
Why not artifact log with the rest of the artifacts?

**Comment by ShiftyRail** [comment]
@jknpj I think it's that, there is no log_artifact proc as far as I can tell, and the investigation logs get added in the server logs.

**Pull request merged**